firestore/tests/unit/v1/test_transaction.py

```diff
@@ -329,6 +329,109 @@ def test__commit_failure(self):
             metadata=client._rpc_metadata,
         )
 
+    def _get_all_helper(self, client, document_pbs):
+        # Create a minimal fake GAPIC with a dummy response.
+        firestore_api = mock.Mock(spec=["batch_get_documents"])
+        response_iterator = iter(document_pbs)
+        firestore_api.batch_get_documents.return_value = response_iterator
+
+        # Attach the fake GAPIC to a real client.
+        client._firestore_api_internal = firestore_api
+
+    def test_get_all(self):
+        from google.cloud.firestore_v1.document import DocumentSnapshot
+        import types
+
+        client = _make_client()
+        trans = self._make_one(client)
+        data1 = {"a": u"cheese"}
```

Comment:
Better use `transaction = self._make_one(client)`, to make it consistent with the other test methods.